openfisca_us/tests/policy/baseline/irs/credits/earned_income/earned_income_tax_credit.yaml

```diff
@@ -0,0 +1,65 @@
+# OpenFisca-US test file derived from the f19.its.csv and f19.ots.csv files
```

Comment:
Could you move these to the taxsim folder, or rename it `integration.yaml`? So far we've been using tests named after variables for unit tests.